test/juttled/juttled.spec.js

```diff
@@ -85,7 +85,8 @@ describe("Juttled Tests", function() {
             return start_job_check_job_id();
         });
 
-        it("Start 2 jobs at once, should return both jobs", function() {
+        it("Start 2 jobs at once, should return both jobs", function(done) {
+            this.timeout(5000);
```

Comment:
What's this for? Think you're shooting yourself in the foot here as 5s is not long enough.